monai/handlers/mlflow_handler.py

```diff
@@ -222,6 +249,36 @@ def _set_experiment(self):
             raise ValueError(f"Cannot set a deleted experiment '{self.experiment_name}' as the active experiment")
         self.experiment = experiment
 
+    @staticmethod
+    def _get_pandas_dataset_info(pandas_dataset):
+        dataset_name = pandas_dataset.name
+        return {
+            f"{dataset_name}_digest": pandas_dataset.digest,
+            f"{dataset_name}_samples": pandas_dataset.profile["num_rows"],
+        }
+
+    def _log_dataset(self, sample_dict: dict[str, Any], context: str | KeysCollection = "train") -> None:
```

Comment:
I don't think the typehint here for context can be `KeysCollection`, what do you think?

Reply:
This type hints was added to solve the mypy error, since the input parameter in line 466 is `KeysCollection`. While currently MLFlow only supports string type key, I can update all the relative parameters to `str` and remove the `KeysCollection`.

Thanks,
Bin